charts/caddy-ingress-controller/Chart.yaml

```diff
@@ -4,7 +4,7 @@ home: https://github.com/caddyserver/ingress
 description: A helm chart for the Caddy Kubernetes ingress controller
 icon: https://caddyserver.com/resources/images/caddy-circle-lock.svg
 type: application
-version: 1.2.0
+version: 1.2.1
```

Comment:
This is a new feature, so I'll like to bump as minor release instead of patch release, @taraszka WDYT?

Reply:
Sounds good to me, let me bump it to 1.3.0 then

Reply:
done